ui/components/Queue.js

```diff
@@ -154,6 +170,9 @@ const fieldComponents = {
       </div>
     )
   },
+  name: ({ job }) => {
+    return job.name
+  },
```

Comment:
Note that I added a new column for the name.
With BullMQ, it is mandatory to set a name for the job...

Don't know if I should add it to Bull3 or not @vcapretz ?

Reply:
yeah, if it works with Bull3 it's fine! 😄

Reply:
Ok, I just handled the case in Bull3 where there was no name so that it doesn't show `__default__` but just `--`